<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoppingCartItemWgt</class>
 <widget class="QWidget" name="shoppingCartItemWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>76</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>76</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>76</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>shoppingCartItemWgt</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_{
border-bottom:1px dashed #00947E;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>26</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_icon">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:#AAAAAA;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_name">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#EEEEEE;</string>
        </property>
        <property name="text">
         <string>巧克力星冰乐巧克力星冰乐</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_standard">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#DADADA;</string>
        </property>
        <property name="text">
         <string>中杯</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_price">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#00D5CF;</string>
        </property>
        <property name="text">
         <string>¥17.56</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="btn_reduce">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/masterDesktop/Resources/order_minus.png);
}
QPushButton:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_num">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>14</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#EEEEEE;
border-top:1px solid #2EB3AF;
border-bottom:1px solid #2EB3AF;</string>
          </property>
          <property name="text">
           <string>20</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_add">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/masterDesktop/Resources/order_add.png);
}
QPushButton:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="masterDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
